src/Controls/src/Core/Layout/Layout.cs

```diff
@@ -58,7 +59,17 @@ protected override void InvalidateMeasureOverride()
 				child.InvalidateMeasure();
 			}
 		}
-		
+
+		internal override ReadOnlyCollection<Element> LogicalChildrenInternal
```

Comment:
@hartez Can we add this code for now so that logic that utilizes the `LogicalChildrenInternal` will work correctly? For example `OnBindingContextChanged` on Element uses this to propagate the `BindingContext` down to the children. `Element_Styles` uses this to propagate down changes to style sheets